qcodes/loops.py

```diff
@@ -263,7 +263,7 @@ def with_bg_task(self, task, end_task=None, min_delay=1):
             end_task: A callable object with no parameters. This object will be
                 invoked at the end  of the measurement loop.
 
-            min_delay (default 1): The minimum number of seconds to wait
+            min_delay (default 0.01): The minimum number of seconds to wait
```

Comment:
I had originally set this to 1 because anything shorter caused the loop to slow appreciably when using matplot. With qtplot it's not a problem. I'm fine with being more aggressive with the default, but I think in the doc string perhaps we should suggest that users lengthen the min_delay if the background task is doing a lot of processing, such as updating a matplot graph.